README.md

```diff
@@ -49,7 +49,7 @@ as ``ovirt_engine_setup_answer_file_path``.
 | ovirt_engine_setup_use_remote_answer_file | False             | If `True`, use answerfile's path on the remote machine. This option should be used if the installation occure on the remote machine and the answerfile is located on there also. |
 | ovirt_engine_setup_update_setup_packages | False              | If `True`, setup packages will be updated before `engine-setup` will be executed. Makes sense if Engine is already installed. |
 | ovirt_engine_setup_perform_upgrade    | False                 | If `True` this role is used to perform upgrade. |
-| ovirt_engine_setup_update_all_packages | True                 | If `True`, all packages will be updated before `engine-setup` will be executed. |
+| ovirt_engine_setup_update_all_packages | True                 | If `True`, all packages will be updated before `engine-setup` will be executed. If `ovirt_engine_setup_offline` is `True` the packages will not be updated. |
```

Comment:
I would rephrase to use a positive case. I.e. explain that `ovirt_engine_setup_offline needs to be False otherwise it's not truly updated